Controller/SSOController.php

```diff
@@ -179,13 +179,13 @@ private function promoteUser(User $user)
     {
         $dm = $this->get('doctrine_mongodb.odm.document_manager');
         $permissionProfileService = $this->get('pumukitschema.permissionprofile');
-        $ldapSerive = $this->get('pumukit_ldap.ldap');
```

Comment:
Since you are changing this, you could use the opportunity to refactor the name to `$ldapService`